yumi_description/launch/yumi.rviz

```diff
@@ -1,236 +0,0 @@
-Panels:
```

Comment:
if they are the same, yes, then there is no need to add new ones ;) But of course it's nice to provide a proper customized `.rviz` file, but I guess, that's fine for now.